pkg/3scale/amp/manual-templates/amp/build.yml

```diff
@@ -11,12 +11,12 @@ objects:
     annotations:
     labels:
       app: zync
-    name: ruby-25-ubi7
+    name: ruby-27-ubi8
   spec:
     tags:
     - from:
         kind: DockerImage
-        name: registry.redhat.io/ubi7/ruby-25
+        name: registry.access.redhat.com/ubi8/ruby-27
```

Comment:
An interesting note about this PR is that it seems that the URL for it is now in `registry.access.redhat.com` domain instead of `registry.redhat.io`. It seems that depending on the image they've changed it to `registry.access.redhat.com`

Reply:
`registry.redhat.io/ubi8/ruby-27` also exists.